cypress/integration/e2e/end_to_end.spec.ts

```diff
@@ -0,0 +1,90 @@
+import Login from '../../pageObjects/login'
+import Purchase from '../../pageObjects/purchase'
+import Signin from '../../pageObjects/signin'
+import Address from '../../pageObjects/address'
+import Orders from '../../pageObjects/orders'
+
+describe('End to End Scenario', function()  {
+   
+  const login = new Login();
+  const purchase = new Purchase();
+  const signin = new Signin();
+  const address = new Address();
+  const orders = new Orders();
+
+  let user: any
+  before(function () {
+      cy.fixture('user').then((data) => {
+          user = data
+      })
+  })
+
+  it('Visit BrowserStack Demo Website', () => {
```

Comment:
This is a mocha framework used. Usually in mocha, Every Test Suite is described in Describe and within describe multiple its means multiple tests. Here in e2e tests, when you run e2e spec file, it is running it nine times so when it will be reported it will be 9 tests but logically it is just one test, recommend to use a single it here

Reply:
resolved on call

Reply:
E2e will be considered as a scenario with the sequence of tests(it)

Reply:
this will be sequence of tests in e2e scenario